src/utils/rfq_client.ts

```diff
@@ -124,7 +124,7 @@ export class RfqClient {
     public async getV2PricesAsync(request: RfqtV2Request): Promise<RfqtV2Price[]> {
         try {
             const response = await this._axiosInstance.post(`${this._rfqApiUrl}/internal/rfqt/v2/prices`, request, {
-                timeout: RFQT_REQUEST_MAX_RESPONSE_MS * 2,
+                timeout: RFQT_REQUEST_MAX_RESPONSE_MS * 3,
                 headers: {
```

Comment:
nit/optional: is there a reason for not changing `RFQT_REQUEST_MAX_RESPONSE_MS` itself?  (it seems like something that can trip people off)

Reply:
It'd be great to revisit this param later. I've made this change to account for seemingly frequent timeouts I encountered during testing, but this may be irrelevant in prod.